csv-provider/signalsFmsRecording.csv.zip.license

```diff
@@ -1,18 +0,0 @@
-# SPDX-FileCopyrightText: 2023 Contributors to the Eclipse Foundation
```

Comment:
Should we add such a license file (signalsFmsRecording.csv.license) to the CSV file because the CSV-file cannot contain comment fields for a potential header?

Reply:
It should already be in the code base ...

Reply:
True, my bad